package.json

```diff
@@ -79,7 +79,7 @@
     "@metamask/eth-sig-util": "4.0.0",
     "@rainbow-me/react-native-animated-number": "0.0.2",
     "@rainbow-me/react-native-payments": "1.1.5",
-    "@react-native-community/async-storage": "1.12.1",
+    "@react-native-async-storage/async-storage": "1.17.7",
```

Comment:
Nit: does this need to be bumped in this PR?

Reply:
Hey! We will likely have to update it since we probably shouldn't use two instances of the same library, it can have some weird conflicts, and segment already lists the new one with new package name as a dev dependency. There's one more thing we need to verify with the changes, the rest are bug fixes and one breaking change that affects tests, it changes how the values are merged.